doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -41,10 +41,18 @@ Output types::
       #  server: 127.0.0.1
       #  port: 6379
       #  async: true ## if redis replies are read asynchronously
-      #  mode: list ## possible values: list|lpush (default), rpush, channel|publish
+      #  mode: list ## possible values: list|lpush (default), rpush, channel|publish, xadd|stream
       #             ## lpush and rpush are using a Redis list. "list" is an alias for lpush
       #             ## publish is using a Redis channel. "channel" is an alias for publish
-      #  key: suricata ## key or channel to use (default to suricata)
+      #             ## xadd is using a Redis stream. "stream" is an alias for xadd
+      #  key: suricata ## string denoting the key/channel/stream to use (default to suricata)
+      #  stream-maxlen: 10000000      ## Automatically trims the stream length to at most
```

Comment:
Seems large default a default? Maybe 100000?

Reply:
unbounded sounds kind of bad, what effects are there if we keep growing?

Reply:
In that case the Redis server will have to handle the size of the stream, e.g. by evicting items once memory fills up. IMHO it is up to the admin of the setup (who knows the access pattern) to maintain a safe stream size -- similar to how a Redis list (`redis.mode: list`) can also grow indefinitely if nothing is popped from it.

However, to be on the safe side I am going to change this setting to be 100K as default and leave it to the user to make it larger or set it to unbounded.

Reply:
What will suri do if this limit is reached?

Reply:
> What will suri do if this limit is reached?

Nothing. When using max len, Redis will evict the oldest data.

Reply:
Correct.

Reply:
Ok, thanks.